watcher.go

```diff
@@ -76,7 +78,8 @@ func (watch *Watch) Go(opts ...Option) {
 	watch.outis.Go(func() error {
 		var (
 			childContext, childContextCancelFunc = context.WithCancel(context.Background())
-			ctx                                  = &Context{
+			ctx                                  = &ContextImpl{
+				id:                ID(strconv.FormatInt(rand.Int63(), 10)),
```

Comment:
Sugiro usar essa função que cria um uuid para ser usado aqui

```go
// NewRequestID create a new id (RID)
func NewRequestID() string {
	var u [16]byte
	if _, err := rand.Read(u[:16]); err != nil {
		panic(err)
	}

	u[8] = (u[8] | 0x80) & 0xBf
	u[6] = (u[6] | 0x40) & 0x4f

	return fmt.Sprintf("%x", u[10:])
}
```


```suggestion
				id:                ID(strconv.FormatInt(rand.Int63(), 10)),
```

Reply:
Criei a issue: https://github.com/Brisanet/outis/issues/20